modules/shared/components/molecules/ImgWithInfo/ImgWithInfo.tsx

```diff
@@ -9,44 +9,114 @@ import { handleImgWithInfoVariant } from '../../../logic/imgWithInfoVariant/imgW
 const ImgWithInfo = ({
   classes,
   children,
+  isResponsive,
 }: IImgWithInfo.IProps): ReactElement => {
-  const boxClasses = classNames(styles.box, classes);
-  return <div className={boxClasses}>{children}</div>;
+  const infoClasses = classNames(
+    styles['info-box'],
+    {
+      [styles['responsive-info-box']]: isResponsive,
+    },
+    classes,
+  );
+  return <div className={infoClasses}>{children}</div>;
 };
 
 const Image = ({
   classes,
-  avatarSize = 'medium',
+  ImageSize,
   ImageVariant,
   profilePic,
   onAvatarClickHandler,
   isHidden = false,
   children,
-}: IImgWithInfo.IProps): ReactElement => {
-  const ImageClasses = classNames(styles['header-wrapper'], classes);
+  isResponsive,
+}: IImgWithInfo.IImage): ReactElement => {
+  const ImageClasses = classNames(
+    {
+      'w-10 h-10': ImageSize === 'medium',
+      'w-5xl h-5xl': ImageSize === 'super-large',
+    },
+    {
+      [styles['responsive-image']]: isResponsive,
+      'mr-4': !isResponsive,
+    },
+    classes,
+  );
+
   return (
     <div className={ImageClasses}>
-      <div className={styles.header}>
-        {ImageVariant === 'avatar' && (
-          <Avatar
-            size={avatarSize}
-            variant={handleImgWithInfoVariant(isHidden, profilePic)}
-            profilePic={profilePic}
-            onClick={onAvatarClickHandler}
-          />
-        )}
-      </div>
+      {ImageVariant === 'avatar' && (
+        <Avatar
+          size={ImageSize}
+          variant={handleImgWithInfoVariant(isHidden, profilePic)}
+          profilePic={profilePic}
+          onClick={onAvatarClickHandler}
+        />
+      )}
       {ImageVariant === 'icon' && children}
     </div>
   );
 };
 
-const Info = ({ classes, children }: IImgWithInfo.IProps): ReactElement => {
-  const bodyClasses = classNames(classes);
-  return <div className={bodyClasses}>{children}</div>;
+const Info = ({
+  classes,
+  children,
+  isResponsive,
+}: IImgWithInfo.IProps): ReactElement => {
+  const infoWrapperClasses = classNames(
+    styles['info-wrapper'],
+    {
+      [styles['responsive-info-wrapper']]: isResponsive,
+    },
+    classes,
+  );
+  return <div className={infoWrapperClasses}>{children}</div>;
+};
+
+const Title = ({
+  classes,
+  children = '',
+  titleSize,
+}: IImgWithInfo.ITitle): ReactElement => {
+  const text = children.length !== 0 ? children : 'Anonymous';
+
+  const titleClasses = classNames(
+    styles.title,
+    {
+      'text-sm': titleSize === 'small',
+      'text-base': titleSize === 'medium',
+      'text-lg': titleSize === 'large',
+    },
+    classes,
+  );
+  return <h3 className={titleClasses}>{text}</h3>;
```

Comment:
We agreed on adding `title` property so that when the user hover over it he can see the full question.